### PR TITLE
fix(insights): update insights example for Vue InstantSearch

### PR DESCRIPTION
**Pull request by eunjae-lee**
## Summary

This PR updates the insights example for Vue InstantSearch.
It's using the temporary build from CodeSandbox, so this shouldn't be merged until we release the latest version of Vue InstantSearch.

[preview](https://codesandbox.io/s/github/algolia/doc-code-samples/tree/fix/insights-vue/Vue%20InstantSearch/algolia-insights?file=/public/index.html)

### CODE REVIEW

**Review by tkrugg — CHANGES_REQUESTED** [review]
All good, other than the init call needs to be restored if my understand is correct

File: Vue InstantSearch/algolia-insights/public/index.html [hunk 1]
```diff
@@ -7,13 +7,12 @@
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title>Algolia Insights</title>
     <script>
-      !function(e,a,t,n,s,i,c){e.AlgoliaAnalyticsObject=s,e.aa=e.aa||function(){(e.aa.queue=e.aa.queue||[]).push(arguments)},i=a.createElement(t),c=a.getElementsByTagName(t)[0],i.async=1,i.src="https://cdn.jsdelivr.net/npm/search-insights@0.0.14",c.parentNode.insertBefore(i,c)}(window,document,"script",0,"aa");
-    
-      // Initialize library
-      aa('init', {
-        applicationID: 'latency',
-        apiKey: '6be0576ff61c053d5f9a3225e2a90f76'
-      })
```
[the remark this answers is not in the document]
Comment (eunjae-lee) [reply Vue InstantSearch/algolia-insights/public/index.html:16]: No, actually `insights` middleware calls `init`, so it's no longer needed :)

[the remark this answers is not in the document]
Comment (tkrugg) [reply Vue InstantSearch/algolia-insights/public/index.html:16]: Oh I didn't know that. I've tested the code and I see the following error. Is there an examplanation for it?





codesandbox: https://codesandbox.io/s/github/algolia/doc-code-samples/tree/fix/insights-vue/Vue%20InstantSearch/algolia-insights?file=/public/index.html

[the remark this answers is not in the document]
Comment (eunjae-lee) [reply Vue InstantSearch/algolia-insights/public/index.html:16]: I'm not 100% sure, but I feel like it's a bug from CodeSandbox.
When I managed to reproduce the issue, [the constructor of AlgoliaAnalytics](https://github.com/algolia/search-insights.js/blob/master/lib/insights.ts#L127) was called twice. It shouldn't, right?

So, the second call overwrites and the initialization is gone, so we see that error message. I downloaded the project, ran locally and it worked well.

https://user-images.githubusercontent.com/499898/116407189-cceed480-a831-11eb-9ebf-8b0ea0d98100.mp4

I'll check with CodeSandbox support.

[the remark this answers is not in the document]
Comment (eunjae-lee) [reply Vue InstantSearch/algolia-insights/public/index.html:16]: @tkrugg It seems it's fixed on CodeSandbox' side. The link above no longer gives the error.

**Comment by eunjae-lee** [comment]
[Not urgent] Can I have your reviews here? @shortcuts @Haroenv

File: Vue InstantSearch/algolia-insights/public/index.html [hunk 2]
```diff
@@ -7,13 +7,12 @@
...
+      var ALGOLIA_INSIGHTS_SRC = "https://cdn.jsdelivr.net/npm/search-insights@1.7.1";
```
Comment (Haroenv) [inline Vue InstantSearch/algolia-insights/public/index.html:10]: v2 now?

File: Vue InstantSearch/algolia-insights/public/index.html [hunk 3]
```diff
@@ -7,13 +7,12 @@
...
     <title>Algolia Insights</title>
     <script>
-      !function(e,a,t,n,s,i,c){e.AlgoliaAnalyticsObject=s,e.aa=e.aa||function(){(e.aa.queue=e.aa.queue||[]).push(arguments)},i=a.createElement(t),c=a.getElementsByTagName(t)[0],i.async=1,i.src="https://cdn.jsdelivr.net/npm/search-insights@0.0.14",c.parentNode.insertBefore(i,c)}(window,document,"script",0,"aa");
-    
-      // Initialize library
-      aa('init', {
-        applicationID: 'latency',
-        apiKey: '6be0576ff61c053d5f9a3225e2a90f76'
-      })
+      var ALGOLIA_INSIGHTS_SRC = "https://cdn.jsdelivr.net/npm/search-insights@2";
```
Comment (Haroenv) [inline Vue InstantSearch/algolia-insights/public/index.html:10]:
```suggestion
      var ALGOLIA_INSIGHTS_SRC = "https://cdn.jsdelivr.net/npm/search-insights@2.0.0";
```

**Review by Haroenv — COMMENTED** [review]
looks good otherwise, has the doc been updated to match this?

**Comment by eunjae-lee** [comment]
> looks good otherwise, has the doc been updated to match this?

yeah actually the doc is already up-to-date!
https://www.algolia.com/doc/guides/building-search-ui/going-further/send-insights-events/vue/

**Review by Haroenv — APPROVED** [review]

**Pull request merged**